liiatools/annex_a_pipeline/spec/__init__.py

```diff
@@ -17,15 +19,26 @@
 
 SCHEMA_DIR = Path(__file__).parent
 
+region_config = env_config("REGION_CONFIG", cast=str)
```

Comment:
What happens if this config variable is not set.  Do we have a default or should we raise an exception to say that it isn't set and needs to be?

Reply:
I see below that you catch things when trying to load the config file. Better to catch errors early if you can. Leave that there, but maybe have something here as well.

Reply:
We haven't got any error catching for the other environment variables, should we? There shouldn't be a default so therefore should raise an exception

Reply:
We shouldn't just rely on the app crashing if a configuration isn't set correctly and then catching the logs. We want to control errors if possible to prevent escalation if possible. Even though it's missing for the others, ideally I'd prefer if we added further checking for all of them.

Reply:
Yeah that sounds good, have added some error catching around this (and moved to common)

Reply:
I don't think we should have defaults because this config contains data processor instructions, such as which LAs have signed up and agreed to share data. Therefore if this doesn't exist for some reason I don't think we should be processing things as we might not have permission to do so.

Happy to jump on a quick call and talk through this if easier